README.md

```diff
@@ -1 +1,23 @@
-TODO :)
\ No newline at end of file
+This library was designed for gathering HAR files from requests sent using `axios`
+
+# Usage
+
+In order to use this package import it by 
+```
+import { AxiosHarTracker } from './axios-har-tracker'
```

Comment:
Example should show how to use the package outside of this repo so in import it should be package name, not relative package path

Reply:
true